### PR TITLE
ENH: added a print statement

### PR DESCRIPTION
**Pull request by nrwslac**

### CODE REVIEW

File: project_name/tests/conftest.py [hunk 1]
```diff
@@ -0,0 +1 @@
+print("Hello World")
```
Comment (ZLLentz) [inline project_name/tests/conftest.py:1]: :wave:

File: project_name/tests/conftest.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+# print a smiley face
+print("\U0001f600")
```
Comment (klauer) [inline project_name/tests/conftest.py:3]: 😄

Comment (klauer) [reply project_name/tests/conftest.py:3]: Give this a try - assuming your editor supports it:

```suggestion
print("😄")
```

Comment (tangkong) [reply project_name/tests/conftest.py:3]: VSCode and pylance complains about this for me, but  maybe it works for you

Comment (klauer) [reply project_name/tests/conftest.py:3]: No warnings on my end - and it runs in my terminal:

image

What pylance error (warning?) is raised?

Comment (tangkong) [reply project_name/tests/conftest.py:3]: Revisiting this the print statement was actually fine.  It was trying to assign a value to the emoji that complained, understandably.  (I forget why I expected this to work)
image

**Pull request closed without merging**